plugins/check_nt.c

```diff
@@ -341,7 +341,7 @@ int main(int argc, char **argv){
 
 		2) If the counter you're going to measure is percent-based, the code will detect
 		 the percent sign in its name and will attribute minimum (0%) and maximum (100%)
-		 values automagically, as well the �%" sign to graph units.
+		 values automagically, as well the ¨%" sign to graph units.
```

Comment:
U+000A8 doesn't make any sense to me here. I  just vote for U+00022 (")
```suggestion
		 values automagically, as well the "%" sign to graph units.
```

Reply:
Looks reasonable...